project/ReleaseSettings.scala

```diff
@@ -40,12 +40,16 @@ object ReleaseSettings {
     autoScalaLibrary := false,
   )
 
+  lazy val scalaReleaseSettings = releaseSettings ++ Seq(
+    crossPaths := true,
+    releaseCrossBuild := true,
+  )
+
   lazy val releaseSettings = Seq(
```

Comment:
could this be made private. so that no one uses this directly?

Reply:
good call, added